classes/main.rb

```diff
@@ -0,0 +1,122 @@
+require './rental'
+require './person'
+require './teacher'
+require './student'
+require './book'
```

Comment:
- [ ] Please remember to use `require_relative` instead of `require`, `require_relative` is used for files that are on the same folder, kindly fix this in order to check the functionality of the app. 😄
    - Please change **ALL** the `require` on your project for `require_relative`